OpenRA.Game/Translation.cs

```diff
@@ -34,17 +37,30 @@ public TranslationReferenceAttribute(params string[] requiredVariableNames)
 
 	public class Translation
 	{
-		readonly IEnumerable<MessageContext> messageContexts;
+		readonly Dictionary<CultureInfo, FluentBundle> contexts = new Dictionary<CultureInfo, FluentBundle>();
```

Comment:
This seems to only ever be used with `CultureInfo.Invariant`, so do we need this as a dictionary at all?

Reply:
I have to admit that I didn't test it with any other locale than `en` yet.

Reply:
Changing system local won't impact it because the code doesn't offer a way to use anything but `CultureInfo.Invariant`.

Reply:
I mean I am not yet sure what this `CultureInfo` is for: input or output. If it is input which is my first guess than this has to be `Invariant` to be compatible with OpenRA.

Reply:
This has not been resolved - either we need this as a dictionary and the fact that we only ever put one culture into it is a bug, or there is a chunk of redundant code that can be removed!

Reply:
Err sorry, it seems you pushed an update that removes it.